Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -416,7 +416,6 @@ extension PlaceCarouselViewController: PlacesProviderDelegate {
     }
 
     func placesProvider(_ controller: PlacesProvider, didError error: Error) {
-        showErrorMessageIfNoPlaces()
```

Comment:
I'm not sure if I understand when `didError` should be called.

If we have no network, we shouldn't be able to get to firebase, so will be reliant of cached values.

If we have network, but the server is down, then we should be able to get to firebase.

In that case, I'm not sure we should be doing anything with `didError` and we should definitely not be setting `isUpdating = false`.

Probably add a comment to that effect in `PlacesProvider`.